src/lib.rs

```diff
@@ -12,6 +12,7 @@ generate_macros!();
 
 mod admin;
 mod config;
+pub mod migrations;
```

Comment:
Make private and add a public re-export of `Migrator`?

Reply:
The `test_utils` is still necessary for other crates.

Reply:
I see.  Maybe parts of it could be moved into trussed at some point.  I think the FS test utils could be useful in other crates too.